DelvEdit/src/com/interrupt/dungeoneer/editor/EditorApplication.java

```diff
@@ -634,12 +636,17 @@ public void draw() {
 		renderer.editorIsRendering = player == null;
 		renderer.enableLighting = showLights;
 
-		level.fogStart = 500f;
-		level.fogEnd = 500f;
-		level.viewDistance = 500f;
+        if (!showLights) {
+            GlRenderer.fogStart = 500f;
+            GlRenderer.fogEnd = 500f;
+            GlRenderer.fogColor.set(Color.BLACK);
```

Comment:
Would it make sense to abstract this to a new object like `EditorLevelInfo` that we can use to hold any default values we want to use for editor levels? Or hold a reference to a `LevelInfo` object that stores editor settings? Could we even make this one accessible in the editor?

Are there any other default values that are using in other places?

Reply:
There is no such thing as an editor level?

That code basically lets the user see the entire level unobstructed from lighting/fog.

Reply:
Yes, I know. I mean a level displayed in the editor, right.